test/e2e/lifecycle/cluster_upgrade.go

```diff
@@ -182,6 +182,65 @@ var _ = SIGDescribe("etcd Upgrade [Feature:EtcdUpgrade]", func() {
 	})
 })
 
+var _ = SIGDescribe("kube-proxy migration [Feature:KubeProxyDaemonSetMigration]", func() {
+	f := framework.NewDefaultFramework("kube-proxy-ds-migration")
+
+	// Create the frameworks here because we can only create them
+	// in a "Describe".
+	testFrameworks := createUpgradeFrameworks()
+
+	Describe("Upgrade kube-proxy from static pods to a DaemonSet", func() {
+		It("should maintain a functioning cluster [Feature:KubeProxyDaemonSetUpgrade]", func() {
```

Comment:
If this only works on GCE, you should skip unless it is GCE.

Reply:
Sure, done.